bin/generate

```diff
@@ -3,7 +3,7 @@
 require_relative '../lib/helper'
 require 'generator'
 
-paths = Generator::Paths.new(track: EXERCISM_RUBY_ROOT, metadata: METADATA_REPOSITORY_PATH)
+paths = Generator::Paths.new(track: EXERCISM_RUBY_ROOT, docs: EXERCISM_RUBY_DOCS, metadata: METADATA_REPOSITORY_PATH)
```

Comment:
`docs:` is not an appropriate argument here, the docs path should be derived from the `track: EXERCISM_RUBY_ROOT`